src/test-helpers/queries.ts

```diff
@@ -0,0 +1,17 @@
+import { findAllByText, screen } from '@testing-library/dom'
+
+/**
+ * Gets a thought by value.
+ */
+export const findThoughtByText = async (value: string, container?: HTMLElement) => {
+  const thoughtNodes = await (container ? findAllByText(container!, value, { exact: true }) : screen.findAllByText(value, { exact: true }))
+  return thoughtNodes.find(t => t.hasAttribute('contenteditable'))
+}
```

Comment:
I prefer separate modules for each exported function. Having the module name be the same as the function name makes it easier to know where functions live, and it also make for very fast access using your editor's quick file find feature.